### PR TITLE
update anatomy content for all components + clean up image paths

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Updated the anatomy content for all components as well as small content updates here and there.
Also cleaned up the anatomy + preview image paths a little bit.

### CODE REVIEW

File: src/_data/hint.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 module.exports = {
   en : {
-    heading: "Include hint text and error message text for task success",
+    heading: "Include a hint text for task success",
```
Comment (ethanWallace) [inline src/_data/hint.js:3]: This heading is weird to me. Could it be something like "Include hint text for better task success"?

File: src/en/components/footer/code.md [hunk 2]
```diff
@@ -39,15 +42,22 @@ Choose the **full display** if you need to include:
...
-- Set the `contextual-links`, by passing an object or an object in a string in the following format.
+- For the contextual band, set the `contextual-links`, by passing an object or an object in a string in the following format:
+```js
+{
+“1st link label”: “href”
+“2nd link label”: “href”
+“3rd link label”: “href”
+}
+```
+- For the main band, set the `sub-links`, by passing an object or an object in a string in the following format:
```
Comment (ethanWallace) [inline src/en/components/footer/code.md:53]: "For the main band" should be "For the footer links band"

**Review by ethanWallace — COMMENTED** [review]
Looking good! Just two small comments

File: src/_data/hint.js (hunk 1 above)
Comment (melaniebmn) [reply src/_data/hint.js:3]: Good question, will have to check with Amy since she wrote this.

File: src/en/components/footer/code.md (hunk 2 above)
Comment (melaniebmn) [reply src/en/components/footer/code.md]: Updated

**Comment by melaniebmn** [comment]
@ethanWallace updated the footer band content - for the other content I need to check with Amy. Do you want me to keep this PR open until then or can I merge now and correct that sentence next week after talking to Amy?

**Review by ethanWallace — APPROVED** [review]
LGTM! We can go back an edit the other sections next week

**Pull request merged**